Makefile

```diff
@@ -127,6 +127,9 @@ DUNESTRAPOPT=--root .
 # We regenerate always as to correctly track deps, can do better
 # We do a single call to dune as to avoid races and locking
 _build/default/theories_dune _build/default/ltac2_dune .dune-stamp: FORCE
+ifneq ($(filter -split,$(COQ_DUNE_EXTRA_OPT)),) # avoid depending on local coq-core
```

Comment:
Maybe instead of changing the dependencies of _build/default/theories_dune depending on if we are split or not, we should have a separate target _build/default/theories_dune_split? The makefile can then pick which one it builds and copies out of _build.

Reply:
changed